diff --git a/node_modules/gatsby-plugin-theme-ui/src/provider.js b/node_modules/gatsby-plugin-theme-ui/src/provider.js
index 6d4162f..810b52c 100644
--- a/node_modules/gatsby-plugin-theme-ui/src/provider.js
+++ b/node_modules/gatsby-plugin-theme-ui/src/provider.js
@@ -1,4 +1,4 @@
-import { ThemeProvider, merge } from 'theme-ui'
+import { ThemeUIProvider, merge } from 'theme-ui'
 import { useThemedStylesWithMdx } from '@theme-ui/mdx'
 import React from 'react'
 
@@ -23,13 +23,13 @@ const Root = ({ children }) => {
   const fullTheme = merge(themeWithPrism, localTheme)
 
   return (
-    <ThemeProvider theme={fullTheme}>
+    <ThemeUIProvider theme={fullTheme}>
       <MDXProvider
         components={useThemedStylesWithMdx(useMDXComponents(components))}
       >
         {children}
       </MDXProvider>
-    </ThemeProvider>
+    </ThemeUIProvider>
   )
 }
 
